BUILDER_CONTAINER.md

```diff
@@ -7,24 +7,21 @@
 ### GitHub OAuth Application
 
 1. Create a GitHub Organization
-1. [Setup an OAuth application](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/registering-oauth-apps) for your GitHub organization.
+1. [Setup a GitHub application](https://github.com/settings/apps/new) for your GitHub organization.
 1. Set the value of `Homepage URL` to `http://${APP_HOSTNAME}`
-1. Set the value of `Authorization callback URL` to `http://${APP_HOSTNAME}/#/sign-in`
-1. Record the the client-id and client-secret. These will be used for the `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET` build variables (seen below).
+1. Set the value of `User authorization callback URL` to `http://${APP_HOSTNAME}/#/sign-in`
```

Comment:
The `redirect_uri` parameter in the `href` for the "Sign In with GitHub" button is `http:///${APP_HOSTNAME}/`.  Is `.../#/sign-in` working as well?

Reply:
good catch